net.nokyan.Resources.json

```diff
@@ -0,0 +1,139 @@
+{
+    "id": "net.nokyan.Resources",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "45",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-nightly",
+        "org.freedesktop.Sdk.Extension.llvm16"
+    ],
+    "command": "resources",
+    "finish-args": [
+        "--device=dri",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=full",
+        "--env=RUST_LOG=resources=debug",
+        "--env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/usr/share:/run/host/share:/var/lib/flatpak/exports/share:~/.local/share/flatpak/exports/share",
+        "--filesystem=/var/lib/flatpak/app:ro",
+        "--filesystem=/var/lib/flatpak/exports/share:ro",
+        "--filesystem=xdg-data/flatpak/app:ro",
+        "--filesystem=xdg-data/flatpak/exports/share:ro",
+        "--filesystem=host",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.Flatpak"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-nightly/bin:/usr/lib/sdk/llvm16/bin",
+        "env": {
+            "CARGO_REGISTRIES_CRATES_IO_PROTOCOL": "sparse",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/app/bin/mold",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/app/bin/mold"
+        },
+        "test-args": [
+            "--socket=x11"
+        ]
+    },
+    "modules": [
+        {
+            "name": "glib",
```

Comment:
*groan*

if you need to have a custom glib, then it's broken.

Reply:
It's the same patch used by MenuLibre (https://github.com/flathub/org.bluesabre.MenuLibre) because GLib within flatpak has trouble recognizing applications

Reply:
two wrongs don't make a right. That's for sure.

Reply:
Why do you need the patch? I don't see the application managing any desktop entries.

Reply:
Ah, seems like it is for the applications tab?

Reply:
It doesn't require a patched GLib anymore.